MAINTAINERS.md

```diff
@@ -0,0 +1,143 @@
+| Paticipate in project's Milestones planning                                                                                                 |            ✗            |            ✗           |            ✓            |
+| Can propose new RFC or<br> participate in review of existing RFC                                                                            |            ✓            |            ✓           |            ✓            |
+| Can request rework of RFCs<br> in represented area of responsibility                                                                        |            ✗            |            ✓           |            ✓            |
+| Can request rework of RFCs<br> in any part of the project                                                                                   |            ✗            |            ✗           |            ✓            |
+| Can manage release process of the project                                                                                                   |            ✗            |            ✗           |            ✓            |
+| Can represent the project in public as a Maintainer                                                                                         |            ✗            |            ✗           |            ✓            |
+
+These roles are merit based. Refer to the corresponding section for specific
+requirements and the nomination process.
+
+## Contributor
+
+A Contributor invests time and resources to improve oneTBB project.
+Anyone can become a Contributor by bringing value in one of the following ways:
+  * Answer questions from community members.
+  * Submit feedback to design proposals.
+  * Review and/or test pull requests.
+  * Test releases and report bugs.
+  * Contribute code, including bug fixes, features implementations,
+and performance optimizations.
+  * Contribute design proposals.
+
+## Code Owner
+
+A Code Owner has responsibility for a specific project component or a functional
+area. Code Owners are collectively responsible, with other Code Owners,
+for developing and maintaining their component or functional areas, including
+reviewing all changes to corresponding areas of responsibility and indicating
+whether those changes are ready to be merged. Code Owners have a track record of
+contribution and review in the project.
+
+Requirements:
+  * Track record of accepted code contributions to a specific project component.
+  * Track record of contributions to the code review process.
+  * Demonstrated in-depth knowledge of the architecture of a specific project
+    component.
+  * Commits to being responsible for that specific area.
+
+The process of becoming a Code Owner is:
+1. A Contributor is nominated by opening a PR modifying the MAINTAINERS.md file
+including name, Github username, and affiliation.
+2. At least two specific component Maintainers approve the PR.
```

Comment:
Well, there is only one Maintainer right now in `oneTBB Core` component, for example. How is it possible for "at least two" Maintainers to ACK such PR? :)

Reply:
Also it states earlier that Code Owners "participate in promotions of Code Owners", how do they participate if that cannot approve these.  I think maybe "specific component Maintainer" -> "Code Owners of that component or Maintainers"

Reply:
That was an error, code-owners cannot participate in promotion process